Automattic/wp-calypso
# WooCommerce: Split out post-form styles

This PR splits out the post-form styles into a separate file, so our root stye.scss doesn't grow unmanageable. No other changes are made -- just moving styles.

To Test:
* Go to `http://calypso.localhost:3000/store/products/:site/add`
* Make sure things like the name and description inputs are aligned correctly.

cc @kellychoffman @jameskoster
